docs/cluster-agent/README.md

```diff
@@ -29,19 +29,29 @@ The goal of the DCA is to enhance the experience of monitoring Kubernetes:
 * It acts as a proxy between the API server and the node agent in order to separate concerns.
 * It provides cluster level metadata that can only be found in the API server to the node agents for them to enrich the metadata of the locally collected metrics.
 * It enables a cluster level collection of data such as the monitoring of services or SPOF and events that could otherwise than via a mix of [Leader Election] and [Autodiscovery] could not be monitored.
+* It implements the External Metrics Provider interface and can be registered as such in Kubernetes.
+
 
 ## When to use the DCA
 
 Beyond a few hundred nodes hitting the API Server, can surface a non negligible impact.
 We recommend using the DCA should you want to alleviate the impact of the agents on your infrastructure and continue getting the same experience.
 Furthermore, should you want to isolate the node agent to the node, reducing the RBAC rules to solely read metrics and metadata from the kubelet.
 
+Should you want to leverage the Horizontal Pod Autoscaling feature of Kubernetes you can use the DCA to pull metrics from Datadog.
+You will be able to autoscale your deployments based off of any metric available in your Datadog account.
+Please refer to the HPA section of the technical documentation if you want to read more about this feature.
```

Comment:
no "please", add a link to the section in question

`Refer to the [HPA section](link here) of the technical documentation to read more about this feature.`